src/androidTest/java/androidx/core/graphics/ImageDecoderTest.kt

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (C) 2018 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package androidx.core.graphics
+
+import android.content.Context
+import android.graphics.Bitmap
+import android.graphics.ImageDecoder
+import android.support.test.InstrumentationRegistry
+import org.junit.After
+import org.junit.Assert.assertEquals
+import org.junit.Before
+import org.junit.Test
+import java.io.ByteArrayOutputStream
+import java.io.File
+
+class ImageDecoderTest {
+
+    @Before fun before() {
+        val bStream = ByteArrayOutputStream().apply {
+            Bitmap
+                .createBitmap(1, 1, Bitmap.Config.ARGB_8888)
+                .compress(Bitmap.CompressFormat.JPEG, 100, this)
+        }
+
+        context.openFileOutput(TEST_FILE.name, Context.MODE_PRIVATE).use {
```

Comment:
Could you just use the ByteBuffer variant of ImageDecoder.createSource instead to avoid relying on the file system during testing?

Reply:
Fixed!